integration/samples/fail-missing-deps/package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "non-peer-deps",
+  "name": "fail-missing-deps",
   "description": "https://github.com/ng-packagr/ng-packagr/pull/687",
   "dependencies": {
     "foobar": "1.0.0"
   },
   "ngPackage": {
     "dest": "./dist",
+    "deleteDestPath": true,
```

Comment:
Can you explain why this is needed?

Reply:
When working on writing `fail*` integration sample tests, I've noticed that it's best to have a clean slate every time the test was executed. But now that I've removed the other more problematic tests, we can probably revert this one change.

P.S. I still have 2 new `fail` tests on my local machine, the fixes I've tried break the watch mode so I've postponed them for a later PR. One of those tests if for testing cleanup on builds failing because of missing imports inside secondary entry points. - the current code doesn't delete the destination.